### PR TITLE
PR Template

### PR DESCRIPTION
**Pull request by mukul-seagate11**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
-<!--- Provide a general summary of your changes in the Title above -->
 
-## Description
-<!--- Describe your changes in detail -->
+### Description
```
Comment (shailesh-vaidya) [inline .github/pull_request_template.md:2]: May be we can add here "Describe your change in brief"

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
-## Related Issue
-<!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
-<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
+### Related Issue
+ - [ ] If suggesting a new feature or change, please discuss it in an issue first
```
Comment (shailesh-vaidya) [inline .github/pull_request_template.md:5]: Should we ask "If you are proposing new change please raise issue first and then raise PR"

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+ - [ ] If fixing a bug, there should be an issue describing it with steps to reproduce
+ - [ ] Please link to the issue here.
 
-## Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here. -->
+### Motivation and Context
+ - [ ] Why is this change required? What problem does it solve?
```
Comment (shailesh-vaidya) [inline .github/pull_request_template.md:10]: This should be tracked in issue or at both issue/PR

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
+ - [ ] How your change affects other areas of the code, etc.
 
-## Screenshots (if appropriate):
+### Screenshots (if appropriate)
+
+### Checklist
+ - [ ] tested locally
+ - [ ] added new dependencies
+ - [ ] updated the docs
+ - [ ] added a test
```
Comment (shailesh-vaidya) [inline .github/pull_request_template.md:24]: May be we can add  `Codacy` and `Sign off` checks.

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya, suggested commented incorporated, can you approve

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**